src/cascadia/TerminalApp/Commandline.cpp

```diff
@@ -80,11 +30,6 @@ char** Commandline::BuildArgv()
 // - <none>
 void Commandline::AddArg(const std::wstring& nextArg)

```

Comment:
I'm worried about the mingling of wide and narrow strings in this class. Why's this one wide, but all the arguments it stores narrow?

Reply:
I'll add a doc comment to the file for clarity